db/migrations/versions/initial_mirgration.py

```diff
@@ -0,0 +1,53 @@
+"""empty message
+
+Revision ID: initial_migration
+Revises:
+Create Date: 2022-07-15 12:23:02.0
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "initial_migration"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "Code",
+        sa.Column("id", sa.String(length=20), nullable=False),
+        sa.Column("description", sa.Text(), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "Event",
+        sa.Column(
+            "id",
+            sqlalchemy_utils.types.uuid.UUIDType(binary=False),
+            nullable=False,
+        ),
+        sa.Column("code", sa.String(length=20), nullable=False),
+        sa.Column("entity_identifier", sa.Text(), nullable=False),
+        sa.Column("timestamp", sa.DateTime(), nullable=False),
+        sa.Column("auditor_user_id", sa.Text(), nullable=True),
```

Comment:
Should this just be 'user_id' not sure who the _auditor_ would be in this context?

Reply:
yes i think this was changed from user_id originally but I will change back because i agree were already in the context of the audit service so no need to add auditor